_posts/2018-06-15-planungen-fuer-das-xcamp-laufen-auf-hochtouren.md

```diff
@@ -0,0 +1,26 @@
+---
+layout:     post
+title:      Planungen für das XCamp laufen auf Hochtouren
+authorPage: Jens Bothmer
+published:  15.06.2018
+image:      media/XCamp-Team_2018_1280px-700x441.jpg
+tags:       XCamp2018
+---
+
+Der 25. August 2018 steht fest. Die Location ist im 
+![Skydeck des Silberturms bei der DB Systel GmbH](https://skydeck.deutschebahn.com/) atemberaubend. 
+Erste Sessions sind bei uns eingereicht. Wir, das ![Orga-Team](https://xcamp.co/barcamp-for-agile-management/ueber/) 
```

Comment:
Here we have the exclamation mark problem again. It's not an image but a link!

Reply:
ok!